### PR TITLE
Added XLM-Roberta large and LaBSE

### PR DESCRIPTION
**Pull request by x-tabdeveloping**
Added multilingual base line models.

### CODE REVIEW

**Comment by x-tabdeveloping** [comment]
@KennethEnevoldsen as per #81 we might want to consider using `CLS` pooling instead of mean for XLM-R models, what do you think?

**Comment by KennethEnevoldsen** [comment]
I would just load in with sentence transformer (we just want the naive defaults here). I believe it is mean, which I also suspect will do better. Especially as the CLS token is used during pre-training XLMR as I recall.

**Comment by x-tabdeveloping** [comment]
Okay then we just keep it as is. Can you review the code real quick @KennethEnevoldsen ?

File: src/seb/registered_models/hf_models.py [hunk 1]
```diff
@@ -285,12 +285,48 @@ def create_xlm_roberta_base() -> EmbeddingModel:
         embedding_size=768,
     )
 
+    # Beware that this uses mean pooling currently, and we might want to change it to CLS in the future
```
Comment (KennethEnevoldsen) [inline src/seb/registered_models/hf_models.py:288]: LaBSE would use the CLS no?

**Review by KennethEnevoldsen — APPROVED** [review]

File: src/seb/registered_models/hf_models.py (hunk 1 above)
Comment (x-tabdeveloping) [reply src/seb/registered_models/hf_models.py:288]: It does, yes

**Pull request merged**